config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -93,9 +93,8 @@ presubmits:
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
   - name: pull-cluster-api-provider-azure-e2e-windows
     path_alias: "sigs.k8s.io/cluster-api-provider-azure"
-    optional: true
+    optional: false
     decorate: true
-    always_run: false
```

Comment:
I wasn't sure on this setting but the `pull-cluster-api-provider-azure-e2e` e2e job didn't have it.

Reply:
we want to run for all PRs? or only when we call this job?

if we want to run on demand we need this setting

Reply:
we want this to run on all PRs

Reply:
so this is good